src/test/java/rx/plugins/RxJavaPluginsTest.java

```diff
@@ -108,6 +108,14 @@ public void testObservableExecutionHookViaRegisterMethod() {
     }
 
     @Test
+    public void testSingleExecutionHookViaRegisterMethod() {
```

Comment:
If so then I believe it would be better to leave it as is and fix all test methods in bulk when we prepare IDE settings and SCA tools. What do you think?